spec/lib/ember_cli/runner_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "ember_cli/runner"
+
+describe EmberCli::Runner do
+  describe "#run!" do
+    context "when the command fails" do
+      it "writes STDERR and STDOUT to `err`" do
+        out = StringIO.new
+        err = StringIO.new
+        runner = EmberCli::Runner.new(err: err, out: out)
+
+        expect { runner.run!("echo 'hello'; exit 1") }.
```

Comment:
This test isn't currently enforcing the capture of the command's STDERR, as stated in `it`. Perhaps something like:

```ruby
runner.run!("echo 'hello'; echo 'world' > /dev/stderr; exit 1")
...
expect(err.read).to match(/hello\nworld/)
expect(out.read).to eq("hello\nworld\n")
```